doorway-ui-components/src/page_components/help-center/DoorwayLinkableCardGroup.scss

```diff
@@ -15,7 +23,3 @@
     padding: var(--doorway-standard-spacing);
   }
 }
-
```

Comment:
Wasn't used since spelling didn't match. Correcting the spellings leads to undesirable formatting